### ISSUE TITLE
Good use-case examples for all objects

### ISSUE THREAD

**Issue by sc0ttkclark** [sc0ttkclark/wordpress-fields-api#1]
I'd like help on fleshing out the examples (what fields to register for each use-case) on each of the main objects:

* [ ] Customizer
* [ ] User profile

Additional examples for long-term reach:

* [ ] Settings screens
* [ ] Post editor

**Comment by sc0ttkclark** [comment]
If you want to help here, you can do a PR on the `README.md` or provide a text-based list of object+fields+options on this thread.

**Comment by sc0ttkclark** [comment]
There may also be a use for an `/examples/` folder, with `.php` files for each use-case, and the simplest forms for reference shown in `README.md`, the more in-depth examples in the `.php` files.

**Comment by KingYes** [comment]
What's about custom fields for Menus screen?

**Comment by helenhousandi** [comment]
Fair warning about menus: that screen currently sends ALL of the fields on the screen (including hidden ones, which are only hidden visually) as POST data, which currently causes serious problems with large menus.

**Comment by sc0ttkclark** [comment]
I have hit this same wall with too many fields being submitted to PHP because of the default [max_input_vars](http://php.net/manual/en/info.configuration.php#ini.max-input-vars).

In the past, I've had to use JS to create JSON objects for each "row", in this case that'd be menu items. Anyways, still a significant hurdle to get past for Menu items.

**Comment by celloexpressions** [comment]
Menus will no longer be an issue in a matter of weeks, hopefully :) I think we might actually wait on allowing any field extensibility there in the meantime so that we can leverage this when it's ready.

http://github.com/voldemortensen/menu-customizer/

I'd also love to be able to use this when creating widgets, not sure about the feasibility of that though.

**Comment by sc0ttkclark** [comment]
Closing in favor of #22
